Dockerfile

```diff
@@ -52,13 +48,17 @@ RUN curl --location https://github.com/kubernetes-sigs/kustomize/releases/downlo
         chmod +x /usr/local/bin/kustomize
 
 # Install Google Cloud SDK
-RUN curl -O https://dl.google.com/dl/cloudsdk/channels/rapid/downloads/google-cloud-sdk-${CLOUD_SDK_VERSION}-linux-x86_64.tar.gz && \
-        tar xzf google-cloud-sdk-${CLOUD_SDK_VERSION}-linux-x86_64.tar.gz && \
-        rm google-cloud-sdk-${CLOUD_SDK_VERSION}-linux-x86_64.tar.gz && \
+RUN curl --location https://dl.google.com/dl/cloudsdk/channels/rapid/downloads/google-cloud-sdk-${CLOUD_SDK_VERSION}-linux-x86_64.tar.gz --output google-cloud-sdk.tar.gz && \
+        tar xzf google-cloud-sdk.tar.gz && \
+        mv google-cloud-sdk /usr/bin/google-cloud-sdk && \
+        ls /usr/bin/google-cloud-sdk && \
+        chmod +x /usr/bin/google-cloud-sdk/bin && \
         ln -s /lib /lib64 && \
         gcloud config set core/disable_usage_reporting true && \
         gcloud config set component_manager/disable_update_check true && \
-        gcloud config set metrics/environment github_docker_image && \
-        gcloud --version
```

Comment:
Maybe it would be a good idea to keep the versions in Dockerfile so it's output into the build log and ensure they are executable. Not sure though. The commands are:

```
docker --version
terraform --version
kubectl version --client -o yaml
yq --version
kustomize version